admin/check/check_crypto_inc.php

```diff
@@ -52,8 +52,8 @@
 
 if( config_get_global( 'login_method' ) != LDAP ) {
 	check_print_test_warn_row(
-		'login_method is set to MD5',
-		config_get_global( 'login_method' ) == MD5,
-		'MD5 password encryption is currently the strongest password storage method supported by MantisBT.'
+		'login_method is set to SAFE_HASH',
```

Comment:
Thanks for the heads up, I missed those as I only looked at the code and omitted the comments.

Note that I'm still working on updating the admin checks, that will be covered with an upcoming commit.